sys/vane/jael.hoon

```diff
@@ -704,33 +865,45 @@
     ::
         %pubs
       %-  curd  =<  abet
-      (~(pubs ~(feed su our.tac urb sub etn) hen) who.tac)
+      (~(pubs ~(feed su hen our.tac urb sub etn sap) hen) who.tac)
     ::
     ::  seen after breach
     ::    [%meet our=ship who=ship]
     ::
         %meet
-      %+  cure  our.tac
-      [[%meet who.tac]~ urb]
+      %^  cure  hen  our.tac
+      [[%meet who.tac life.tac pass.tac]~ urb]
+    ::
+    ::  XX should be a subscription
+    ::  XX reconcile with .dns.eth
+    ::  request domains
+    ::    [%turf ~]
+    ::
+        %turf
+      ::  ships with real keys must have domains,
+      ::  those with fake keys must not
+      ::
+      ?<  =(fak.own.sub ?=(^ tuf.own.sub))
```

Comment:
Wait huh, so this will crash if a fakenet has DNS configured? I get we don't actually want to *use* it, but it shouldn't crash just because there's data there, should it?

Reply:
The current fake-key design derives data as needed, and would be messed up by actually receiving ethereum events. We could instead ignore or clear this data if set (here, or in %ames, or in vere), but it really just shouldn't be set imo.